lib/trento/users/policy.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Trento.Users.Policy do
+  @moduledoc """
+  Policy for the User resource
+
+  User with the ability all:all or users:all can perform any operation on the users.
+  """
+  @behaviour Bodyguard.Policy
+
+  alias Trento.Users.User
+
+  def authorize(action, %User{} = user, User) when action in [:index, :show],
+    do: has_read_ability?(user)
+
+  def authorize(action, %User{} = user, User) when action in [:update, :delete, :create],
+    do: has_write_ability?(user)
+
+  def authorize(_, _, _), do: false
+
+  defp has_write_ability?(user), do: has_global_ability?(user) or has_users_all_ability?(user)
+  defp has_read_ability?(user), do: has_global_ability?(user) or has_users_all_ability?(user)
```

Comment:
`has_read_ability?` should be on top of `has_write_ability?`

Reply:
done